os/kernel/init/os_start.c

```diff
@@ -111,6 +112,10 @@ extern const uint32_t g_idle_topstack;
  * Pre-processor Definitions
  ****************************************************************************/
 
+/* This is the set of all CPUs */
+
+#define SCHED_ALL_CPUS		((1 << CONFIG_SMP_NCPUS) - 1)
```

Comment:
CONFIG_SMP_NCPUS is valid when the SMP is enabled only. Is this ok when SMP is disabled?

Reply:
We have set it to 1 in default case. 

```
printf("#ifndef CONFIG_SMP_NCPUS\n");
printf("#  define CONFIG_SMP_NCPUS 1\n");
```